args/args.go

```diff
@@ -33,6 +33,7 @@ const (
 	kernelCmdlineDemo         = "clri.demo"
 	kernelCmdlineLog          = "clri.loglevel"
 	kernelCmdlineHighContrast = "clri.hc"
+	kernelMediaCheck          = "clri.mediacheck"
```

Comment:
Since this value is needed in the iso utils code, perhaps it should be exported and used in isoutils/isoutils.go for consistency. 
```suggestion
	KernelMediaCheck          = "clri.mediacheck"
```

Reply:
Done